scaaml_intro/train.py

```diff
@@ -11,39 +11,43 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Model training."""
 
 import argparse
 import json
+import sys
 from termcolor import cprint
+
 import tensorflow as tf
 from tensorflow.keras.callbacks import ModelCheckpoint, TensorBoard
-from scaaml.utils import tf_cap_memory
 import tensorflow.keras.backend as K
-from scaaml.utils import get_model_stub
+
 from scaaml.intro.generator import create_dataset
 from scaaml.intro.model import get_model
+from scaaml.utils import get_model_stub
 from scaaml.utils import get_num_gpu
+from scaaml.utils import tf_cap_memory
 
 
 def train_model(config):
     tf_cap_memory()
-    TRAIN_GLOB = "datasets/%s/train/*" % config['algorithm']
-    TEST_GLOB = "datasets/%s/test/*" % config['algorithm']
-    TEST_SHARDS = 256
-    NUM_TRACES_PER_TEST_SHARDS = 16
-    BATCH_SIZE = config['batch_size'] * get_num_gpu()
+    TRAIN_GLOB = f"datasets/{config['algorithm']}/train/*"  # pylint: disable=C0103
```

Comment:
We should rename these instead of disabling the linter.
These are not file-level nor class-level constants and should therefore not be capitalized